consumer-services/src/main/scala/za/co/absa/spline/consumer/service/repo/AttributeRepositoryImpl.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package za.co.absa.spline.consumer.service.repo
+
+import com.arangodb.async.ArangoDatabaseAsync
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Repository
+import za.co.absa.spline.consumer.service.model.search.FoundAttribute
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Repository
+class AttributeRepositoryImpl @Autowired()(db: ArangoDatabaseAsync) extends AttributeRepository {
+
+  import za.co.absa.spline.persistence.ArangoImplicits._
+
+  def findAttributesByPrefix(search: String)(implicit ec: ExecutionContext): Future[Array[FoundAttribute]] = {
+    val result = db.queryStream[FoundAttribute](
+      """
```

Comment:
Let's use margins (`|`) in the multi-line string. It makes the code easier to read (the boundary between the inner and outer code is clearly visible), and it also makes it easier to handle indents when copy/pasting or auto-formatting.

Reply:
nit: indents. The commons style for AQL is 4-space tab.